scrapers.js

```diff
@@ -2048,6 +2048,91 @@ let scrapers = [
uid\":\"BS\",\"name\":\"Bahamas\"},{\"uid\":\"VU\",\"name\":\"Vanuatu\"},{\"uid\":\"FK\",\"name\":\"Ilhas Malvinas\"},{\"uid\":\"GM\",\"name\":\"Gâmbia\"},{\"uid\":\"QA\",\"name\":\"Catar\"},{\"uid\":\"JM\",\"name\":\"Jamaica\"},{\"uid\":\"CY\",\"name\":\"Chipre\"},{\"uid\":\"PR\",\"name\":\"Porto Rico\"},{\"uid\":\"PS\",\"name\":\"Palestina\"},{\"uid\":\"BN\",\"name\":\"Brunei\"},{\"uid\":\"TT\",\"name\":\"Trinidad e Tobago\"},{\"uid\":\"CV\",\"name\":\"Cabo Verde\"},{\"uid\":\"PF\",\"name\":\"Polinésia Francesa\"},{\"uid\":\"WS\",\"name\":\"Samoa\"},{\"uid\":\"LU\",\"name\":\"Luxemburgo\"},{\"uid\":\"RE\",\"name\":\"Ilha da Reunião\"},{\"uid\":\"KM\",\"name\":\"Comores\"},{\"uid\":\"MU\",\"name\":\"Maurício\"},{\"uid\":\"FO\",\"name\":\"Ilhas Faroé\"},{\"uid\":\"MQ\",\"name\":\"Martinica\"},{\"uid\":\"ST\",\"name\":\"São Tomé e Príncipe\"},{\"uid\":\"AN\",\"name\":\"Antilhas Neerlandesas\"},{\"uid\":\"DM\",\"name\":\"Dominica\"},{\"uid\":\"GP\",\"name\":\"Guadalupe\"},{\"uid\":\"TO\",\"name\":\"Tonga\"},{\"uid\":\"KI\",\"name\":\"Quiribati\"},{\"uid\":\"FM\",\"name\":\"Micronésia\"},{\"uid\":\"BH\",\"name\":\"Bahrein\"},{\"uid\":\"AD\",\"name\":\"Andorra\"},{\"uid\":\"MP\",\"name\":\"Ilhas Marianas do Norte\"},{\"uid\":\"PW\",\"name\":\"Palau\"},{\"uid\":\"SC\",\"name\":\"Seicheles\"},{\"uid\":\"AG\",\"name\":\"Antígua e Barbuda\"},{\"uid\":\"BB\",\"name\":\"Barbados\"},{\"uid\":\"TC\",\"name\":\"Turks e Caicos\"},{\"uid\":\"VC\",\"name\":\"São Vicente e Granadinas\"},{\"uid\":\"LC\",\"name\":\"Santa Lúcia\"},{\"uid\":\"YT\",\"name\":\"Mayotte\"},{\"uid\":\"VI\",\"name\":\"Ilhas Virgens Americanas\"},{\"uid\":\"GD\",\"name\":\"Granada\"},{\"uid\":\"MT\",\"name\":\"Malta\"},{\"uid\":\"MV\",\"name\":\"Maldivas\"},{\"uid\":\"KY\",\"name\":\"Ilhas Cayman\"},{\"uid\":\"KN\",\"name\":\"São Cristóvão e Névis\"},{\"uid\":\"MS\",\"name\":\"Montserrat\"},{\"uid\":\"NU\",\"name\":\"Niue\"},{\"uid\":\"PM\",\"name\":\"São Pedro e Miquelão\"},{\"uid\":\"CK\",\"name\":\"Ilhas Cook\"},{\"uid\":\"WF\",\"name\":\"Wallis e Futuna\"},{\"uid\":\"AS\",\"name\":\"Samoa Americana\"},{\"uid\":\"MH\",\"name\":\"Ilhas Marshall\"},{\"uid\":\"AW\",\"name\":\"Aruba\"},{\"uid\":\"LI\",\"name\":\"Liechtenstein\"},{\"uid\":\"VG\",\"name\":\"Ilhas Virgens Britânicas\"},{\"uid\":\"SH\",\"name\":\"Santa Helena\"},{\"uid\":\"JE\",\"name\":\"Jersey\"},{\"uid\":\"AI\",\"name\":\"Anguilla\"},{\"uid\":\"GG\",\"name\":\"Guernsey\"},{\"uid\":\"SM\",\"name\":\"San Marino\"},{\"uid\":\"BM\",\"name\":\"Bermudas\"},{\"uid\":\"TV\",\"name\":\"Tuvalu\"},{\"uid\":\"NR\",\"name\":\"Nauru\"},{\"uid\":\"GI\",\"name\":\"Gibraltar\"},{\"uid\":\"PN\",\"name\":\"Ilhas Pitcairn\"},{\"uid\":\"MC\",\"name\":\"Mônaco\"},{\"uid\":\"VA\",\"name\":\"Vaticano\"},{\"uid\":\"IM\",\"name\":\"Ilha de Man\"},{\"uid\":\"GU\",\"name\":\"Guam\"},{\"uid\":\"SG\",\"name\":\"Singapura\"},{\"uid\":\"SS\",\"name\":\"Sudão do Sul\"},{\"uid\":\"SX\",\"name\":\"São Martinho\"},{\"uid\":\"BL\",\"name\":\"São Bartolomeu\"}]")
```

Comment:
This is very confused, why is there a JSON string here? If you need this data, you can put it on the object above (just like `_ufs`)

Reply:
Yep, the data is needed. It should be manually extracted from website DOM. Because the probability of changes are near zero, is fast than use puppeteer.
The comment is for an eventual change, to clarify how to obtain the data.